templates/zerver/help/restrict-name-and-email-changes.md

```diff
@@ -2,39 +2,50 @@
 
 {!admin-only.md!}
 
-## Prevent users from changing their name
+## Restrict name changes
 
-By default, any user can [change their name](/help/change-your-name). You
-can instead prevent users from changing their name. Organization
-administrators can always [change anyone's name](/help/change-a-users-name).
+By default, any user can [change their name](/help/change-your-name).
+You can instead prevent users from changing their name. This setting is
+especially useful if user names are managed via an external source, and
+synced into Zulip via the [Zulip API](/api) or another method.
```

Comment:
We should probably mention LDAP and link to https://zulip.readthedocs.io/en/latest/production/authentication-methods.html#synchronizing-data here.